src/Components/Authentication/Desktop/ForgotPasswordForm.tsx

```diff
@@ -25,7 +25,7 @@ export class ForgotPasswordForm extends Component<
   }
 
   onSubmit = (values: InputValues, formikBag: FormikProps<InputValues>) => {
-    Grepcaptcha(GrepcaptchaAction.ForgotSubmit)
+    repcaptcha("forgot_submit")
```

Comment:
👍 nice and simple, while also safe. `snake_case` is kinda ugly but /shrug, things are 1:1.

Reply:
Snake case is required by google's API.

Reply:
cool yeah, that's what I figured